.moban.dt/core-travis.yml.jj2

```diff
@@ -0,0 +1,151 @@
+{% extends 'ci/travis.yml.jj2' %}
+
+{% block custom_python_versions %}
+python:
+{% if travis_dist not in ['precise', 'trusty'] %}
```

Comment:
Note that if we find a better solution for https://github.com/coala/coala/issues/6053 on trusty, the old Python 3.4.2 minimum dep will be restored, or maybe before then the need for py34 has evaporated.

Reply:
Template moved to https://gitlab.com/coala/mobans/merge_requests/135 , as coala-quickstart and others need the same fixes